specification/schemas/types/Links.yaml

```diff
@@ -0,0 +1,9 @@
+type: object
```

Comment:
I think this has broken the documentation on the GET /v1/messages/{messageId} endpoint somehow:

![image](https://github.com/NHSDigital/communications-manager-api/assets/879532/90469315-04e9-42a6-b287-7d673851e540)

Its now showing as an `any` with no properties defined on it